### ISSUE TITLE
[C#] Implement new Concept Exercise: resource-lifetime

### ISSUE THREAD

**Issue by mikedamay** [exercism/v3#1641]
This issue describes how to implement the `resource-lifetime` concept exercise for the C# track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

The goal of this exercise is to teach the student how to safely control a resource's lifetime in C#.

## Learning objectives

- Know how to control a resource's lifetime with the `using` statement in C#.

## Concepts

- `resource-lifetime` as it is handled in C#.

## Prerequisites

- `resource-cleanup`

Other prerequisites will depend on the story.

### Hints
[using statement][using-statement] documentation describes regexes and the flavour built into the .NET libraries.

### After

[using statement][using-statement] documentation describes regexes and the flavour built into the .NET libraries.

## Representer

Consider adding a feature to normalize the using statement to the [representer][representer].

## Analyzer

This exercise does not require any specific analyzer logic to be added to the [analyzer][analyzer].

## Implementing

To implement this exercise, please [follow these instructions](https://github.com/exercism/v3/blob/master/languages/csharp/reference/implementing-a-concept-exercise.md).

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.

[using-statement]: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement
[analyzer]: https://github.com/exercism/csharp-analyzer
[representer]: https://github.com/exercism/csharp-representer

**Comment by ErikSchierboom** [comment]
@mikedamay This looks to be a duplicate of #1640. Is that correct?

**Comment by mikedamay** [comment]
Weird.  Closed as duplicate of #1640
